distribution/smarthome/conf/things/demo.things

```diff
@@ -1,3 +1,6 @@
 yahooweather:weather:berlin [ location=638242 ]
 yahooweather:weather:losangeles "Los Angeles" [ location=2442047, refresh=120 ]
 
+magic:color-light:magicColor
```

Comment:
Maybe worth to add some other devices than lights, like e.g. thermostats as well?
Maybe also assign them to a common location, add some labels and custom configurations?

Reply:
The magic bundle will get thermostats with PR #4818, afterwards this can be added to the demo setup. Also if one adds new stuff to the magic bundle we can extend this demo.